glide-core/src/client/value_conversion.rs

```diff
@@ -159,6 +160,21 @@ pub(crate) fn convert_to_expected_type(
             )
                 .into()),
         },
+        ExpectedReturnType::ArrayOfBools => match value {
+            Value::Array(array) => {
+                let array_of_bools = array
+                    .iter()
+                    .map(|v| Value::Boolean(from_owned_redis_value::<bool>(v.clone()).unwrap()))
+                    .collect();
+                Ok(Value::Array(array_of_bools))
+            }
+            _ => Err((
+                ErrorKind::TypeError,
+                "Response couldn't be converted to an array of {boolean}",
```

Comment:
Why do you have braces around "boolean"? That's not the array syntax in Rust, and also reads like "an array of array of boolean"